CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Upcoming
 
+* [FEATURE] Introduce new flag to control logging format (#1475)
+* [FEATURE] Ensure Prometheus Operator container runs as `nobody` user by default (#1393)
+* [BUGFIX] Fix reconciliation of ServiceMonitors and Rules when a single namespace is being watched (#1749)
```

Comment:
We are not touching (reconciling) ServiceMonitors or PrometheusRules. How about `Fix reconciliation of Prometheus StatefulSet due to ServiceMonitor and PrometheusRule when a ...`?